File: Frontend/src/app/DTOs/Draw/DrawCanvasStorageDTO.ts
import { DrawCanvasStoragePointDTO } from "./DrawCanvasStoragePointDTO";

export interface DrawCanvasStorageDTO {
    id: string;
    created: Date;
    deleted: Date | null;
    color: string;
    fromPosistion: DrawCanvasStoragePointDTO;
    toPosition: DrawCanvasStoragePointDTO | null;
    text: string | null;
}

File: Backend/DTOs/Draw/DrawCanvasStorageDTO.cs
namespace Learnz.DTOs;
public class DrawCanvasStorageDTO
{
    public Guid Id { get; set; }
    public DateTime Created { get; set; }
    public DateTime? Deleted { get; set; }
    public string Color { get; set; }
    public DrawCanvasStoragePointDTO FromPosition { get; set; }
    public DrawCanvasStoragePointDTO? ToPosition { get; set; }
    public string Text { get; set; }
}